src/Sylius/Behat/spec/Context/Transform/LexicalContextSpec.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use PhpSpec\ObjectBehavior;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class LexicalContextSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith();
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Behat\Context\Transform\LexicalContext');
+    }
+
+    function it_implements_context_interface()
+    {
+        $this->shouldImplement(Context::class);
+    }
+
+    function it_transforms_price_string_to_integer()
+    {
+        $this->getPriceFromString('10.00')->shouldReturn(1000);
+        $this->getPriceFromString('0.30')->shouldReturn(30);
+        $this->getPriceFromString('0.0134')->shouldReturn(1);
```

Comment:
Are you sure, that this is expected behaviour? Maybe it would be better to throw some exception, that it cannot be casted to integer without losing precision